src/main/java/com/stories/service/StoriesServiceImpl.java

```diff
@@ -26,28 +27,25 @@
 
 	@Autowired
 	private MapperFacade mapperFacade;
-	
+
+	String[] statusArray = { "Ready to Work", "Working", "Testing", "Ready to Accept", "Accepted" };
+
 	@Override
 	public String createStory(StoryDomain storyDomain) throws Exception {
 		StoryModel storyModel = new StoryModel();
```

Comment:
You can refactor one line code here, but it is just a (suggestion).

You could have StoryModel storyModel = mapperFacade.map(storyDomain, StoryModel.class);
instead of:

StoryModel storyModel = new StoryModel();
storyModel = mapperFacade.map(storyDomain, StoryModel.class);